packages/web/src/components/gcds-search/gcds-search.css

```diff
@@ -0,0 +1,70 @@
+@layer defaults, state.focus;
+
+@layer defaults {
+  :host {
+    .gcds-search {
+      .gcds-search__header {
+        clip: rect(1px,1px,1px,1px);
+        height: 1px;
+        margin: 0;
+        overflow: hidden;
+        position: absolute;
+        width: 1px;
+      }
+
+      .gcds-search__form {
+        display: flex;
+        margin: var(--gcds-search-margin) !important;
+      }
+
+      input.gcds-search__input {
+        width: 100%;
+        height: auto;
+        min-height: var(--gcds-search-min-width-and-height);
+        font: var(--gcds-search-font);
+        padding: var(--gcds-search-padding) !important;
+        background-color: var(--gcds-search-default-background);
+        background-image: none;
+        color: var(--gcds-search-default-text);
+        border: var(--gcds-search-border-width) solid currentColor;
+        border-radius: var(--gcds-search-border-radius);
+        border-top-right-radius: 0;
+        border-bottom-right-radius: 0;
```

Comment:
Can you change `border-bottom` to `border-block-end` to be more language inclusive?